clean-language/README-copy.md

```diff
@@ -0,0 +1,146 @@
+They enable us to show instead of tell.
+
+### Changing metaphors changes the way we think
+Did you know that on average we use six metaphors a minute? Surprised? Me too! They are so embedded into our speech that mostly we are unconscious about them.
+Research suggests that metaphors actually change the way we think of a concept on an unconscious level.
+In 2011 there was a study conducted in which two groups of readers have been presented with two articles describing the same crime.
+In the first article the crime was compared to a beast and in another to a disease.
+When asked to solve the crime issue, readers of the first article suggested control strategies while the second group came up with diagnostic or treatment strategies.
+So actually changing the metaphor changes how we think about the issue or an idea at hand.
+
+### Metaphors are capable of changing your experience
+In communication we often fail to see the connection the other person is making in their head.
+So we declare it nonsense.
+It's no wonder that we clash with each other, because we assign different meanings to the same things.
+ 
+In my clean language workshop it was an eye opening experience to observe a magnitude of metaphors people create in response to the same question.
+
+## Clean language and clean questions
+Clean Language is a questioning technique used for discovering, exploring and working with people's own personal metaphors.
+Clean means the questioner only uses the words of the "questionee" without adding their own interpretations.
+This way the language of the questioner is *clean* from their own words.
+Here are two most commonly used questions:
+What kind of ‘x’ is that 'x'?
+Is there anything else about ‘x’?
+(where ‘X’ is a word or phrase that the client has used, e.g "What kind of *problem* is that *problem*?", where problem = X)
+
+## Personal experience with clean language
+With clean language I have come to realization that my own metaphors are not just words I use to describe my experiences.
+They are the views into the models running my life and defining my behavior.
+Recently, my friend and I have been exploring our learning experiences with the help of clean questions.
+Here is how our conversation looked like:
+(Q) When you are learning you are like what?
+(A) I am like a sponge.
+(Q) You are like a sponge, and what kind of sponge is that sponge?
+(A) It is a dry, yellow, hour glass shaped sponge.
+(Q) And it is a dry, yellow, hour glass shaped sponge, is there anything else about that dry?
+(A) It absorbs all the knowledge like water.
+(Q)And when it absorbs all the knowledge what happens next?
+(A) And then it is full and water starts to spill over and it cannot absorb anymore; and then, over time, it dries out...
+(Q) What would you like to have happen?
+(A) I would like my knowledge stay with me instead of drying out.
+And I want to keep learning indefinitely without being limited by saturated sponge.
+
```

Comment:
I love this example so much! It's so powerful. I love it.

Reply:
Thanks!